targets/psa_switch/psa_switch.cpp

```diff
@@ -342,6 +341,21 @@ PsaSwitch::ingress_thread() {
       continue;
     }
 
+    // resubmit - these packets get immediately resub'd to ingress, and skip
+    //            deparsing for now, do not move below multicast or deparse
+    Field &f_resubmit = phv->get_field("psa_ingress_output_metadata.resubmit");
```

Comment:
you can use `auto &` here, and should probably be `const` as well
`const auto &f_resubmit = ...`